test/integration/admin_views_individual_order.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.